Sources/NIOCrashTester/main.swift

```diff
@@ -153,10 +150,11 @@ func main() throws {
         process.arguments = ["_exec", suite, name]
         try process.runProcess()
 
+        process.waitUntilExit()
+        processOutputPipe.closeFile()
```

Comment:
Why did you change this?  Seems unrelated to everything else.

Reply:
@PeterAdams-A it is technically unrelated. The old code however would always hang if the program didn't crash. I discovered that whilst developing this guy so I put it in the same PR :)

Reply:
@PeterAdams-A Please send me an email/Slack ping if you want me to do this in a follow-up.